### PR TITLE
Added a link and removed a stale one

### LINKED ISSUE #19273: Circular / dead link for contributor doc / how to write documentation
### MDN URL

https://developer.mozilla.org/en-US/docs/MDN/Community/Issues

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/MDN/Community/Issues#make_progress_not_noise

### What information was incorrect, unhelpful, or incomplete?

Two links are actually empty: https://github.com/mdn/content/blob/main/files/en-us/mdn/community/issues/index.md?plain=1#L63

### What did you expect to see?

Links to the corresponding pages

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

This page contains other "TBD/ToDo"-ish annotations.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/mdn/community/issues`
* MDN URL: https://developer.mozilla.org/en-US/docs/MDN/Community/Issues
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mdn/community/issues/index.md
* Last commit: https://github.com/mdn/content/commit/52e0541d468a4db057d86b0542b52a34ff1d6aba
* Document last modified: 2022-08-06T02:41:20.000Z

</details>

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Related issues

Fixes #19273 

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Abhiram-Gullapalli** [comment]
Included a link to one and removed a link since it did not have any reference in the doc regarding how to.

File: files/en-us/mdn/community/issues/index.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ As a rule, do:
 
 - Use [GitHub discussion](https://github.com/mdn/mdn-community/discussions) before filing an issue. This helps to keep issues focused and productive.
 - Ask questions using other mechanisms like [chat rooms](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or [forums](<(https://discourse.mozilla.org/c/mdn/236)>) if you are not sure whether something is useful or have a simple question.
-- Read the [contributor documentation]() and [how to write documentation]() first to try to solve the issue yourself.
+- Read the [contributor documentation](https://developer.mozilla.org/en-US/docs/MDN/Community/Contributing) and first to try to solve the issue yourself.
```
Comment (dipikabh) [inline files/en-us/mdn/community/issues/index.md:63]:
```suggestion
- Read our [contribution guidelines](https://developer.mozilla.org/en-US/docs/MDN/Community) and [writing guidelines](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines) first to try to solve the issue yourself.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @Abhiram-Gullapalli, thanks for catching this. I have made the change to reflect what the MDN team intended to say here.

**Review by dipikabh — APPROVED** [review]
Thank you!

**Pull request merged**